crm_claim_code/models/crm_claim.py

```diff
@@ -29,3 +30,14 @@ def copy(self, default=None):
             default = {}
         default['code'] = self.env['ir.sequence'].get('crm.claim')
         return super(CrmClaim, self).copy(default)
+
+    @api.multi
+    @api.depends('code')
```

Comment:
Again, since v8 it is a computed field called display_name with function name_get().